src/components/nodes/association/index.js

```diff
@@ -11,9 +11,12 @@ export default {
   control: false,
   definition(moddle) {
     return moddle.create('bpmn:Association', {
-      associationDirection: `${ direction.none }`,
+      associationDirection: `${direction.none}`,
```

Comment:
PHPStorm did this. It's not part of our ESLint rules but we should add it: https://eslint.org/docs/rules/template-curly-spacing.

I slightly prefer no spaces, as I find I interpret them as actual spaces which will be rendered when skimming over template strinsgs.

Reply:
We decided to add the rule and disallow spaces.